sale_team_default_values/model/sale_order.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# Author: Guewen Baconnier, Leonardo Pistone
+# Copyright 2014-2015 Camptocamp SA
+
+from openerp import models, api
+PROPAGATE_FIELDS = [
```

Comment:
Using a method that returns the list of fields is better for extension, because with a globals such as this one, if a model does a `PROPAGATE_FIELDS.append('lambda_field')` in an extension addon, it could be executed even if the extension addon is not installed.